miner/src/miner.rs

```diff
@@ -228,6 +228,11 @@ impl MinerService for Miner {
 		queue.find(hash)
 	}
 
+	fn pending_transactions(&self) -> Vec<SignedTransaction> {
```

Comment:
"Pending transaction" is a common name for transactions on the network. e.g. http://etherscan.io/txsPending